library/sysroot/Cargo.toml

```
@@ -23,7 +24,7 @@ system-llvm-libunwind = ["std/system-llvm-libunwind"]
 panic-unwind = ["std/panic_unwind"]
 panic_immediate_abort = ["std/panic_immediate_abort"]
 optimize_for_size = ["std/optimize_for_size"]
-profiler = ["std/profiler"]
+profiler = ["profiler_builtins"]
```

```suggestion
profiler = ["dep:profiler_builtins"]
```

to prevent unintentionally adding a feature called `profiler_builtins`.